main.py

```diff
@@ -28,22 +45,24 @@ def get_user_input():
 
 
 if __name__ == '__main__':
-    """
-      Command to run this script:
-
-      $python3 main.py 'https://www.scoopwhoop.com/
-                              entertainment/memes-from-dd-ramayan/' 5
-      $python3 main.py 'https://www.scoopwhoop.com/
-                              entertainment/memes-from-dd-ramayan/'
 
-      For more help: python3 main.py -h
-    """
-    url, maximum_pages = get_user_input()
+    _url, maximum_pages = get_user_input()
 
-    _stampifier = Stampifier(url, maximum_pages)
+    _stampifier = Stampifier(_url, maximum_pages)
 
     try:
-        generated_stamp = _stampifier.stampify()
-        print(generated_stamp)
+        stampifier_output = _stampifier.stampify()
+
+        stamp_file = stampifier_output.stamp_title.replace(' ', '_') \
+            + HTML_EXTENSION
+
+        if stampifier_output.stamp_html:
```

Comment:
You can remove this if condition of that we can throw error and show error UI.

Reply:
My bad! Missed this before merging! Will consider this in future iterations. Thanks for suggestion!